classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,1215 @@
+		i=3;
+	}
+	public void method49(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method50(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method51(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method52(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method53(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method54(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method55(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method56(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method57(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method58(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method59(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method60(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method61(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method62(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method63(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method64(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method65(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method66(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method67(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method68(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method69(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method70(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method71(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method72(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method73(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method74(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method75(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method76(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method77(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method78(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method79(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method80(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method81(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method82(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method83(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method84(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method85(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method86(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method87(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method88(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void method89(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.